### ISSUE TITLE
Outstanding Gitcoin Grants Round 6 Issues

### ISSUE THREAD

**Issue by ceresstation** [gitcoinco/web#6865]
**Describe the bugs**

- [x] Canadian numbers not supported (generic consumer number with major carrier): https://gyazo.com/5d349c5ed4f94a56319d880252d41cab

- [x] First grant on page does not have "add to cart" option on Chrome, Windows 10:
https://gyazo.com/622cf1499a09c006612d8229a29c6dcf

- [x] Match total on link text is incorrect, should read $175k: 
https://gyazo.com/b299b25aaae85ab8d0d52b75c5aa6b7c

- [x] Unable to change grants from other categories to the BLM category in admin: https://gyazo.com/bc1dc31d2db6f9e756b7ea7aa069899f

- [ ] We display a massive number of grants in some cases at the top of the page for bulk checkout, should shorten this to the first 5-10 and then note "and x other grants": https://twitter.com/thegrifft/status/1272526773826224130

@vs77bb @owocki anything I missed here? Happy to provide more details.

**Comment by owocki** [comment]
[ ] absurdly high match esteimates on  https://bits.owocki.com/QwulKRKe https://gitcoin.co/grants/25/lighthouse-ethereum-20-client

**Comment by owocki** [comment]
@ceresstation " First grant on page does not have "add to cart" option on Chrome, Windows 10:" is because youre a team member of that grant.

**Comment by owocki** [comment]
[ ] match amounts are too high https://bits.owocki.com/JruLbblN
